src/kete_core/src/spice/spk_segments.rs

```diff
@@ -63,16 +63,6 @@ impl From<SpkSegmentType> for DafArray {
     }
 }
 
-impl From<i32> for Frame {
```

Comment:
These are removed as these are SPICE specific values and are in conflict with the kete frame conversion. We are using different integers to represent frames.